### DISCUSSION TITLE
How I can save and load custom property, that user not must to see and change?

### DISCUSSION THREAD

**Discussion by Always-prog** [apache/superset#21117] [Q&A / Help] [answered]
Hello.
I working on drag and drop columns in pivot table 2.
I realized functionality drag and drop, but, after reloading page order drops. I need to save that order in chart form Data.

**My pivot table 2 files**
![![image](https://user-images.githubusercontent.com/66589759/185156138-7b3acab3-a7f3-4dcb-bec0-b7c591f98a22.png)

I initializing `columnsOrder` variable in **transoformProps.ts**, and passing it to **Pivot Table Chart.tsx**
![![image](https://user-images.githubusercontent.com/66589759/185156809-e60b418e-44cd-400c-a248-e8c43fe390a0.png)

And in **TableRenderer.jsx**, in `this.props` I getting my initialized variable for saving and using order.

How do I save my columns Order to chart, and use it after refresh page or in dashboard?

**Comment by nytai** [comment]
I'm going to covert this to a discussion as it's just Q&A and not actually an issue with superset

**Reply by Always-prog** [reply]
Okay!

**Comment by Always-prog** [comment, marked as answer]
Answer.
I found function that can update property from code.
![image](https://user-images.githubusercontent.com/66589759/185617835-98660c69-2305-4bb2-a692-7dad16a5f9c1.png)

Just use `setControlValue(controlName, value)` and it will be updated.
